### PR TITLE
(GH-7) PDK Ruby Wrapper commands

### PR DESCRIPTION
**Pull request by jpogran**
- refactor pdk ruby execution methods
- wrapper cmds

### CODE REVIEW

File: cmd/test/test.go [hunk 1]
```diff
@@ -4,12 +4,9 @@ import (
...
 )
 
-var (
-	debug  bool
-	format string
-)
+var ()
 
-func CreateTestCommand() *cobra.Command {
+func CreateCommand() *cobra.Command {
```
Comment (sanfrancrisko) [inline cmd/test/test.go:9]: Was `CreateTestCommand` causing an issue?

**Review by sanfrancrisko — COMMENTED** [review]
Great job 👍 

Just some typos (they may have been in the original), but otherwise LGTM.

File: cmd/test/test.go (hunk 1 above)
Comment (jpogran) [reply cmd/test/test.go:9]: Renamed to align with all of the other wrapper command method names

**Review by da-ar — APPROVED** [review]
This all seems to work really well. 
I believe that it may be worthwhile revisiting the execute function of pdkshell though and ensure that the exit status of the underlying function is bubbled up? This only came to light due to the additional logging now visible with pdkgo.

**Comment by jpogran** [comment]
@da-ar I agree, but think its outside the current PR's remit. I'll have to look into if `exec` bubbles up return codes at all, IIRC it's just 0 if the command worked. I'll ticket this work

**Pull request merged**